samples/highcharts/demo/combo-timeline/demo.js

```diff
@@ -445,22 +640,7 @@ const options = {
             pointFormat: '{point.name}<br><b>{point.y}</b>',
             valueSuffix: ' employees'
         },
-        data: [
-            { x: Date.UTC(2009, 10, 1), y: 1, name: 'Torstein worked alone', image: 'Torstein' },
-            { x: Date.UTC(2010, 10, 20), y: 2, name: 'Grethe joined', image: 'Grethe' },
-            { x: Date.UTC(2011, 3, 1), y: 3, name: 'Erik joined', image: null },
-            { x: Date.UTC(2011, 7, 1), y: 4, name: 'Gert joined', image: 'Gert' },
-            { x: Date.UTC(2011, 7, 15), y: 5, name: 'Hilde joined', image: 'Hilde' },
-            { x: Date.UTC(2012, 5, 1), y: 6, name: 'Guro joined', image: 'Guro' },
-            { x: Date.UTC(2012, 8, 1), y: 5, name: 'Erik left', image: null },
-            { x: Date.UTC(2012, 8, 15), y: 6, name: 'Anne Jorunn joined', image: 'AnneJorunn' },
-            { x: Date.UTC(2013, 0, 1), y: 7, name: 'Hilde T. joined', image: null },
-            { x: Date.UTC(2013, 7, 1), y: 8, name: 'Jon Arild joined', image: 'JonArild' },
-            { x: Date.UTC(2013, 7, 20), y: 9, name: 'Øystein joined', image: 'Oystein' },
-            { x: Date.UTC(2013, 9, 1), y: 10, name: 'Stephane joined', image: 'Stephane' },
-            { x: Date.UTC(2014, 9, 1), y: 11, name: 'Anita joined', image: 'Anita' },
-            { x: Date.UTC(2014, 10, 27), y: 11, name: ' ', image: null }
-        ]
+        data: getTurnover().concat(moreEmployees)
```

Comment:
Do not concat `moreEmployees`, the newer employees should be added to the HTML table like the others.